### ISSUE TITLE
Management toolbar: Search button for mobile and lists with one item

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5391]
_Note: Probably this issue will be finally fixed in DXP_

**To reproduce the issue you can**:
1. Browse to a item list in DXP and by using an SR in desktop view get by tabbing to the Add button, you will see how the SR reads: "list of **one** item". (bad reading, avoid list of items with only an item)
2. In mobile view or desktop 400% zoom, use the Search button to open the search and use it by keyboard (we lost the focus and the search can be closed by esc or by skipping the component)

**How to fix recommendation**:

For 1.:
- Move the search mobile button out of the list
- When we have only an item in the list we will remove the list and let just the item (avoiding list of one item)

For 2.:
- When we open the search, the focus goes directly to the input
- When search box is opened we create a keyboard bucle
- When we press esc key we close the popup and the focus goes back to the trigger (the search button, magnifying glass icon)
- Add title to the close search since is not clear if we clean the popup or we just are cleaning the input:
- Close icon should be moved out of the input since this buttons closes the search box, not clean the input.

![image](https://user-images.githubusercontent.com/7963804/223704874-bbda43b3-27fc-4048-98a2-603b244177b5.png)

**Comment by marcoscv-work** [comment]
Related to: https://issues.liferay.com/browse/LPS-168787

**Comment by marcoscv-work** [comment]
Related to: https://issues.liferay.com/browse/LPS-168108

**Comment by matuzalemsteles** [comment]
@marcoscv-work we need to move these issues to FI because in DXP Clay's ManagementToolbar is no longer used and it has been replaced by the DXP component https://github.com/liferay-frontend/liferay-portal/tree/master/modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/management_toolbar.

It's worth remembering that Clay's implementation is just markup so we can't implement focus resources for example. What we can do here is just update the documentation by changing the markup.

**Comment by marcoscv-work** [comment]
> 

Thanks @matuzalemsteles 
@carloslancha is working on this in DXP: https://issues.liferay.com/browse/LPS-167561

**Comment by marcoscv-work** [comment]
Closing this one since will be fixed in DXP
